demo/src/store/initialState.js

```diff
@@ -0,0 +1,346 @@
+import snuggled from '../images/snuggled.png';
+import solemn from '../images/solemn.png';
+import spoiled from '../images/spoiled.png';
+import stop from '../images/stop.png';
+import tinyBlep from '../images/tiny_blep.png';
+import tuckedInKitty from '../images/tucked_in_kitty.png';
+import whatYouLookinAt from '../images/what_you_lookin_at.png';
+import wheresPoe from '../images/wheres_poe.png';
+
+
+import airplaneEarsThumbnail from '../images/thumbnails/airplane_ears.png';
+import bellyPetThumbnail from '../images/thumbnails/belly_pet.png';
+import bigYawnThumbnail from '../images/thumbnails/big_yawn.png';
+import caughtThumbnail from '../images/thumbnails/caught.png';
+import curledFeetsiesThumbnail from '../images/thumbnails/curled_feetsies.png';
+import downwardFacingCatThumbnail from '../images/thumbnails/downward_facing_cat.png';
+import eyeBoogiesThumbnail from '../images/thumbnails/eye_boogies.png';
+import fluffyThumbnail from '../images/thumbnails/fluffy.png';
+import handsomeCatThumbnail from '../images/thumbnails/handsome_cat.png';
+import hardToCodeWithThumbnail from '../images/thumbnails/hard_to_code_with.png';
+import kittenThumbnail from '../images/thumbnails/kitten.png';
+import lightOfMyLifeThumbnail from '../images/thumbnails/light_of_my_life.png';
+import litterNoseThumbnail from '../images/thumbnails/litter_nose.png';
+import makinBiscuitsThumbnail from '../images/thumbnails/makin_biscuits.png';
+import mirinThumbnail from '../images/thumbnails/mirin.png';
+import pawsUpThumbnail from '../images/thumbnails/paws_up.png';
+import petMyBellyThumbnail from '../images/thumbnails/pet_my_belly.png';
+import phoneBackgroundThumbnail from '../images/thumbnails/phone_background.png';
+import pinkToeBeanThumbnail from '../images/thumbnails/pink_toe_bean.png';
+import shinyThumbnail from '../images/thumbnails/shiny.png';
+import snoozingThumbnail from '../images/thumbnails/snoozing.png';
+import snoringThumbnail from '../images/thumbnails/snoring.png';
+import snuggledThumbnail from '../images/thumbnails/snuggled.png';
+import solemnThumbnail from '../images/thumbnails/solemn.png';
+import spoiledThumbnail from '../images/thumbnails/spoiled.png';
+import stopThumbnail from '../images/thumbnails/stop.png';
+import tinyBlepThumbnail from '../images/thumbnails/tiny_blep.png';
+import tuckedInKittyThumbnail from '../images/thumbnails/tucked_in_kitty.png';
+import whatYouLookinAtThumbnail from '../images/thumbnails/what_you_lookin_at.png';
+import wheresPoeThumbnail from '../images/thumbnails/wheres_poe.png';
+
+
+/**
+ * Sample data for the application. Each item is stored on the site,
+ * along with the quantity in the user's cart and if it is in the user's
+ * cart. If the item is not in the cart, the quantity is 0, but the converse
+ * is not always true, users that modify how many items are in their cart
+ * will usually not want it to disappear immediately.
+ */
+export const initialState = {
+  items: {
+    '1jef2': {
```

Comment:
If someone wanted to add a new item to this list, how would they go about generating an ID?  Assuming it's not too long, I'd add the explanation to the JSDoc comment for the initialState for posterity.

e..g. "When adding a new item, generate an ID by doing steps x,y,z."